### ISSUE TITLE
kf5completion build failure

### ISSUE THREAD

**Issue by kelteseth** [microsoft/vcpkg#17185]
**Host Environment**
- OS: Windows
- Compiler: Latest MSVC 2019

**To Reproduce**
`./vcpkg install kf5completion`

**Failure logs** 
```
D:\Backup\Code\Qt\krita-vcpkg\buildtrees\kf5completion\src\v5.64.0-0ad46db244.clean\src\klineedit.cpp(504): error C2027: use of undefined type 'QKeyEvent'
D:\Backup\Code\Qt\krita-vcpkg\installed\x64-windows\include\QtWidgets/qwidget.h(75): note: see declaration of 'QKeyEvent'
```

**Additional context**
Adding the missing include as a patch fixed it for me: 

``` diff
diff --git a/src/klineedit.cpp b/src/klineedit.cpp
index 47897ce..6641107 100644
--- a/src/klineedit.cpp
+++ b/src/klineedit.cpp
@@ -41,6 +41,7 @@
 #include <QClipboard>
 #include <QMenu>
 #include <QToolTip>
+#include <QKeyEvent>
 
 KLineEditPrivate::~KLineEditPrivate()
 {

```

**Comment by autoantwort** [comment]
Maybe this is fixed in #13467. But I thing in general this is something that should be fixed upstream :)

**Comment by wrobelda** [comment]
Was fixed upstream in 5.65: https://github.com/KDE/kcompletion/commit/56476607c32f25418d6d91cf11e804e689ee8533

#13467 will have it fixed.

**Comment by kelteseth** [comment]
@wrobelda ok then lets close this issue :) Thanks for all your contributions btw 🎉
